apps/amakrushi/src/components/PhoneView/ChatWindow/Comps.tsx

```diff
@@ -331,19 +332,18 @@ export const RenderComp: FC<any> = ({
       console.log('qwe12:', { content });
       return (
         <>
-          <div
+          {/* <div
             style={{ width: '95px', marginRight: '4px', textAlign: 'center' }}>
-            {/* <Avatar src={botImage} size="md" /> */}
-          </div>
-          <Bubble type="text">
+            <Avatar src={botImage} size="md" />
+          </div> */}
+          <Bubble type="text" style={{margin: 'auto', background: 'none', lineHeight: '2.5', padding: '0'}}>
```

Comment:
don't add many attributes in inline style, either use style files or styled component